assets/src/scss/components/_client-section.scss

```diff
@@ -0,0 +1,240 @@
+section {
+
+    .first-container{
+
+        @media screen and (min-width: $desktop) {
+            padding: 50px 145px 50px 145px;
+        }
+    }
+    .kc-row-container {
+        background-image:url(http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/bg-6.jpg);
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-position: 50%;
+    }
+
+
+    .kc-wrap-columns {
+        background:$client-section-background!important;
+        opacity: .8;
+    }
+
+    .full-update-section {
+        padding: 100px 0px 30px;
+    }
+
+    h2 {
+        font-size: 36px;
+        line-height: 46px;
+        text-align: center;
+        text-transform: uppercase;
+        color: $white;
+
+
+        @media screen and (min-width: $tablet){
+            font-size: 40px;
+            line-height: 50px;
+            margin-bottom: 15px;
+        }
+
+
+        @media screen and (min-width: $desktop) {
+            font-size: 46px;
+            padding-bottom: 20px;
+        }
+    }
+
+    p{  
+        color: $ptag-font;
+        margin: 10px 0;
+        font-family: "Noto Serif",serif;
+        font-style: italic;
+        text-align: center;
+        font-size: 24px;
+        margin-left: 15px;
+
+        @media screen and (min-width: $tablet) {
+
+        }
+    }
+
+    p::after {
+        top: 1.6875rem;
+        display: inline-block;
+        width: 6.25rem;
+        height: 0.1875rem;
+        background: $white;
+        content: "";
+
+        @media screen and (min-width: $tablet) {
+            display: block;
+            width: 8.25rem;
+            margin-left: 315px;
+            margin-top: 20px;
+        }
+
+
+        @media screen and (min-width: $desktop) {
+            margin-left: 43.6875rem;  
+            margin-top: 2.25rem;
+        }
+
+    }
+
+    .image-container {
+
+        @media screen and (min-width: $tablet) {
+            display: flex;
+        }
+    }
+
+
+    .image-container-one {
+        padding-bottom: 5.125rem;
+        margin-left: 15px;
+        margin-right: 15px;
+
+        @media screen and (min-width: $tablet) {
+            display: flex;
+            margin-left: 7px;
+            margin-right: 0px;
```

Comment:
When the px value is zero we dont need to add `px` at the end. so this will be : 
` margin-right: 0;`

Reply:
okay! changes have been updated